geopyspark/geotrellis/render.py

```diff
@@ -0,0 +1,56 @@
+from geopyspark.geotrellis.constants import NEARESTNEIGHBOR, ZOOM
+
+class PngRDD(object):
+    def __init__(self, pyramid, rampname):
+        level0 = pyramid[0]
+        self.geopysc = level0.geopysc
+        self.rdd_type = level0.rdd_type
+        self.layer_metadata = list(map(lambda lev: lev.layer_metadata, pyramid))
+        self.max_zoom = level0.zoom_level
+        self.pngpyramid = list(map(lambda layer: self.geopysc._jvm.geopyspark.geotrellis.PngRDD.asSingleband(layer.srdd, rampname), pyramid))
+
+    @classmethod
+    def makePyramid(cls, tiledrdd, rampname, start_zoom=None, end_zoom=0, resamplemethod=NEARESTNEIGHBOR):
+        reprojected = tiledrdd.reproject("EPSG:3857", scheme=ZOOM)
```

Comment:
1. `resamplemethod` should be changed to `resample_method` so that it's consistent with the rest of library, and so that it's consistent with Python's naming conventions.

2. There should be a check to see if `resample_method` is a known resampling method. Like https://github.com/locationtech-labs/geopyspark/blob/master/geopyspark/geotrellis/rdd.py#L202